### ISSUE TITLE
WFS-unfriendly references to messageMetadata properties

### ISSUE THREAD

**Issue by vog** [aixm/donlon-outdated-#20]
All Donlon features contain references to parts of the the "messageMetadata" which belong to the AIXMBasicMessage but not to the features themselves:

``` XML
<message:AIXMBasicMessage>
    ...
    <aixm:messageMetadata>
        ...
        <gmd:CI_ResponsibleParty id="MessageContact">
        ...
    </aixm:messageMetadata>
    ...
    <message:hasMember>...</message:hasMember>
    <message:hasMember>...</message:hasMember>
    <message:hasMember>...</message:hasMember>
    <message:hasMember>
        <aixm:OrganisationAuthority>
            ...
            <gmd:contact xlink:href="#MessageContact"/>
            ...
        </aixm:OrganisationAuthority>
    </message:hasMember>
    <message:hasMember>...</message:hasMember>
    <message:hasMember>...</message:hasMember>
    <message:hasMember>...</message:hasMember>
    ...
</message:AIXMBasicMessage>
```

This is a problem when inserting into WFS, which will only contain the features (everything inside `<message:hasMember>` elements), but not the message metadata. So these references will be broken inside the WFS.

I see only two solutions so far, which are both unsatisfying:
1. Repeat the contents of "MessageContact" (and all other metadata properties) in all features
2. Insert the whole AIXMBasicMessage as one big feature into WFS, which would make the WFS rather useless, because you can't request individual airspaces, etc. anymore.

**Comment by porosnie** [comment]
I think this represents some very early ideas about how metadata could be used in AIXM data sets In order to make Donlon usable as test data for WFS implementations, I would agree with your first proposal - repeat the contact details and other metadata at the level of each feature, without references to the message metadata.

**Comment by porosnie** [comment]
Could we have the support of someone else for this change, so that we can commit?

**Comment by vog** [comment]
Closed via pull request #21
